pipelines/_ingredients/cpDarkForDefects.yaml

```diff
@@ -0,0 +1,31 @@
+description: cp_pipe DARK calibration construction for defect finding.
```

Comment:
Can  this inherit from the default `cpDark.yaml`?  It'd be safer if we only needed to update one location, and that this contains only the difference from the standard cp dark processing.

Reply:
I originally did that, but I first forgot to remove the `cpDark` part and it got all confused.  So we have to inherit, exclude, and override names and some configs and I thought that was more confusing to follow.  However, I could do that if you think it would be a net positive.

Reply:
If it's more complicated than a simple inherit, then it's fine to leave them separate, but please put a "if you edit this, you must also edit the other one" comments in both places so they can hopefully stay in sync.